src/paramValidator/index.js

```diff
@@ -122,7 +122,15 @@ export default class Validator {
                         no = true;
                     }
                     break;
-
+                case 'string':
+                    if (!utils.isString(value) || !value.length ||
+                        (typeof gt === 'number' && value.length <= param.gt) ||
```

Comment:
`!value.length` can conflict with, for example, `lte`.

Reply:
A valid string can be empty. If we need some size, there are the other params. 
So, I would remove that control and make it:
```
     if (!utils.isString(value) ||
            ... 
```

Reply:
Has removed !value.length